apps/prairielearn/src/models/course.ts

```diff
@@ -12,3 +24,50 @@ export async function selectCourseById(course_id: string): Promise<Course> {
     CourseSchema,
   );
 }
+
+export async function selectAuthorizedCourses({
```

Comment:
This name wasn't chosen with much real thought; maybe `selectCoursesWithStaffAccess` would be better, to mimic the name of the sproc and also be a little more precise about what's going on here?

Reply:
I'd prefer that. "Authorized" could easily be interpreted as "student access" which is definitely not what we mean here. Or `selectStaffCourses()` based on `selectEditableCourses()` below?

Reply:
I renamed things to be more precise and consistent in b3e0ec2.